magento/magento2
# Fix #24713 - Symbol of the Belarusian currency BYR is outdated

### Description (*)

This is fix for #24713. As difference between steps described in issue, `Belarusian Ruble (2000-2016)` should not be the selected currency (this is outdated one), but `Belarusian Ruble` instead. In this PR I replaced all usages of old currency, so valid one would be used as default.

### Fixed Issues (if relevant)

1. magento/magento2#24713: Symbol of the Belarusian currency BYR is outdated

### Manual testing scenarios (*)

1. Login to admin
2. Go to the Stores --> Configuration--> Currency Setup
3. In Currency Options add "Belarusian Ruble" to the Allowed Currencies list
4. Go to the Stores--> Currency Rates
5. Check Belarusian Ruble's currency symbol - it should be equal to `BYN`

### Questions or comments


### Contribution checklist (*)
